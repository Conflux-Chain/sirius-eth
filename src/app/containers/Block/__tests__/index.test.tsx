import React from 'react';

describe('<Blocks />', () => {
  it('should match the snapshot', () => {
    const component = <div>Todo. Blocks</div>;
    expect(component).toMatchSnapshot();
  });
});
