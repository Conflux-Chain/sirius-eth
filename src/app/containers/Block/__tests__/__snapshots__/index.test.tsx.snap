// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blocks /> should match the snapshot 1`] = `
<div>
  Todo. Blocks
</div>
`;
